runelite-client/src/main/java/net/runelite/client/plugins/bronzeman/BronzemanPlugin.java

```diff
@@ -43,7 +43,7 @@
 	name = "Bronze Man Mode",
 	description = "Restrict yourself as an Iron Meme with GE use for items you have unlocked.",
 	tags = {"Bronze", "pve", "restrict", "game", "challenge", "bronzeman", "ironman"},
-	type = PluginType.UTILITY,
+	type = PluginType.MISCELLANEOUS,
```

Comment:
i would suggest giving this a plugin type of GAMEMODE incase we decide to add other gamemode orientated plugins, such as client driven group IM, region locking, left click only gamemode etc.